howtos/about_appliance_id.rst

```diff
@@ -0,0 +1,90 @@
+.. _aboutapplianceid:
+
+About Appliance ID
+==================
+
+During the initial :ref:`webui_setup` stage of a Rockstor install,
+an **Appliance ID** is automatically generated.
+This ID, intended as a Universal Unique Identifier (UUID),
+is an exact copy of the host system's, read motherboard's, response to:
+
+.. code-block:: console
+
+    cat /sys/class/dmi/id/product_uuid
+
+.. note::
+
+    With the majority of systems,
+    this is a genuinely unique reference; and no intervention of any sort is required.
+
+The implied unique-ness is assumed by multiple current Rockstor subsystems, e.g.
+when :ref:`add_appliance` as part of setting up a Share :ref:`sharereplication`.
+And as part of configuring a system to use the :ref:`stable_channel` updates.
+
+.. warning::
+
+    WARNING: some systems, via lax firmware implementations, return a NON-Unique / fake product_uuid.
```

Comment:
WARNING: some systems, via lax firmware implementations, return a NON-Unique/fake product_uuid.

Removing spaces before and after '/' for better readibility.